containers/client/Dockerfile

```diff
@@ -1,11 +1,8 @@
 FROM ghcr.io/diracgrid/diracx/client-base
 
-# Needs to specify how we install diracx
-# * from a local wheel package: PACKAGES_TO_INSTALL=/path/to/diracx.whl
-# * from pypi "diracx==v0.a.b"
-ARG PACKAGES_TO_INSTALL
+ARG EXTRA_PACKAGES_TO_INSTALL
```

Comment:
we should have a default here

Reply:
Just `DIRAC` then?

Reply:
and all the `diracx` modules (not only for this DOckerifle)